README.md

```diff
@@ -347,28 +353,81 @@ Set the message size limit for all users. If set to zero, the size will be unlim
   - ignore => Ignore the failure of this test. Allow other tests to complete. Repeat this test the next time the client connects. This option is useful for testing and collecting statistics without blocking mail.
 
 
-##### REPORT_RECIPIENT
+## Reports
+
+##### PFLOGSUMM_TRIGGER
+
+  Enables a report being sent (created by pflogsumm) on a regular basis.
+  - **none** => No report
+  - daily_cron => Daily report for Yesterday will be sent
+  - logrotate => Full report for period will be sent when logs are rotated
+
+This is a new option. The old REPORT options are still supported for backwards compatibility.
+If this is not set and reports are enabled with the old options, logrotate will be used.
+
+##### PFLOGSUMM_RECIPIENT
+
+  Recipient address for pflogsumm reports if they are enabled.
```

Comment:
would drop the "if they are enabled" (same further down)

Reply:
OK